app/controllers/chefs/ingredients_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Chefs::IngredientsController < ApplicationController
+
+  def index
+    @chef = Chef.find(params[:chef_id])
+  end
```

Comment:
For this `index` action I think it makes more sense to send the chef's ingredients to the view instead of the chef themselves.

```ruby
def index
  @chef = Chef.find(params[:chef_id])
  @ingredients = @chef.chef_ingredients
end
```

The reason I say this is because when we think about what an index action is supposed to be doing, it's supposed to send back a collection of whatever the controller is, which in this case, is ingredients.